### PR TITLE
Create Key Gear Item-Screen: Post-Create animation

### PR DESCRIPTION
**Pull request by hedvigoscar**
This PR contains the animation that should show when transitioning from the Create-Screen to the detail-screen.

It's currently lacking translations and animation assets, but needs to be merged now in order to progress with the feature as a whole.

### CODE REVIEW

File: app/src/main/res/layout/activity_create_key_gear_item.xml [hunk 1]
```diff
@@ -63,12 +64,54 @@
...
+            android:visibility="gone"
+            app:layout_constraintBottom_toTopOf="@id/createdLabel"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintVertical_chainStyle="packed"
+            tools:src="@drawable/ic_fire_green"
+            tools:visibility="visible" />
+
+        <!-- TODO: Use text key in code when available -->
```
Comment (FredrikAreschoug) [inline app/src/main/res/layout/activity_create_key_gear_item.xml]: Looks like you have a text key

File: app/src/main/java/com/hedvig/app/Constants.kt [hunk 2]
```diff
@@ -1,5 +1,8 @@
 package com.hedvig.app
 
-const val BASE_MARGIN = 8
-const val BASE_MARGIN_HALF = BASE_MARGIN / 2
-const val BASE_MARGIN_DOUBLE = BASE_MARGIN * 2
+import com.hedvig.app.util.extensions.dp
+
+val BASE_MARGIN = 8.dp
```
Comment (FredrikAreschoug) [inline app/src/main/java/com/hedvig/app/Constants.kt]: :heart_eyes:

File: app/src/main/java/com/hedvig/app/feature/keygear/ui/createitem/CreateKeyGearItemActivity.kt [hunk 3]
```diff
@@ -108,6 +119,53 @@ class CreateKeyGearItemActivity : BaseActivity(R.layout.activity_create_key_gear
...
+
+        val finalRadius = max(root.width, root.height).toFloat() * 1.1f
+        ViewAnimationUtils.createCircularReveal(
+            postCreate,
+            save.centerX,
+            save.centerY,
+            0f,
+            finalRadius
+        ).apply {
+            duration = 400
```
Comment (FredrikAreschoug) [inline app/src/main/java/com/hedvig/app/feature/keygear/ui/createitem/CreateKeyGearItemActivity.kt]: constant

File: app/src/main/res/layout/activity_create_key_gear_item.xml (hunk 1 above)
Comment (hedvigoscar) [reply app/src/main/res/layout/activity_create_key_gear_item.xml]: You'd think so, but it's actually lies. We have the base text key, but no text keys for the values that must be interpolated into it.

**Pull request merged**